ee/clickhouse/queries/test/test_paths.py

```diff
@@ -92,20 +102,20 @@ def test_paths_start(self):
         response = self.client.get("/api/insight/path/?type=%24pageview&start=%2Fpricing").json()
 
         filter = PathFilter(data={"path_type": "$pageview", "start_point": "/pricing"})
-        response = ClickhousePathBase(team=self.team, filter=filter).run(team=self.team, filter=filter,)
+        response = ClickhousePathsNew(team=self.team, filter=filter).run(team=self.team, filter=filter,)
 
         self.assertEqual(len(response), 5)
 
-        self.assertTrue(response[0].items() >= {"source": "1_/pricing", "target": "2_/", "value": 1}.items())
-        self.assertTrue(response[1].items() >= {"source": "1_/pricing", "target": "2_/about", "value": 1}.items())
-        self.assertTrue(response[2].items() >= {"source": "2_/", "target": "3_/about", "value": 1}.items())
-        self.assertTrue(response[3].items() >= {"source": "2_/about", "target": "3_/pricing", "value": 1}.items())
-        self.assertTrue(response[4].items() >= {"source": "3_/pricing", "target": "4_/help", "value": 1}.items())
+        self.assertTrue(response[0].items() == {"source": "1_/pricing", "target": "2_/", "value": 1}.items())
```

Comment:
assertEquals? :)

Reply:
Thanks, that reminds me, I have no idea why we had the `>=` instead of `==`. Will fix!

Reply:
Postgres version has extra elements, hence `>=` and `assertTrue`

Reply:
assertDictContainsSubset?